e2e/pages/app/badge/badge.testpage.component.html

```diff
@@ -0,0 +1,92 @@
+<div class="badge-container">
+    <!-- icon example no content -->
+    <ul class="element-list">
+        <ng-container *ngFor="let vertPos of verticalPositions">
+            <ng-container *ngFor="let horzPos of horizontalPositions">
+                <li>
+                    <ux-icon name="filter" uxBadge badgeColor="warning" [badgeVerticalPosition]="vertPos"
+                        [badgeHorizontalPosition]="horzPos"></ux-icon>
+                </li>
+            </ng-container>
+        </ng-container>
+    </ul>
+    <hr />
+    <!-- icon example no content with overlap -->
+    <ul class="element-list">
+        <ng-container *ngFor="let vertPos of verticalPositions">
+            <ng-container *ngFor="let horzPos of horizontalPositions">
+                <li>
+                    <ux-icon name="list" uxBadge badgeColor="partition9" [badgeVerticalPosition]="vertPos"
+                        [badgeHorizontalPosition]="horzPos" [badgeOverlap]="true" badgeSize="small"></ux-icon>
+                </li>
+            </ng-container>
+        </ng-container>
+    </ul>
+    <hr />
+    <!-- size examples - using icons -->
+    <ul class="element-list">
+        <li *ngFor="let size of sizes">
+            <ux-icon name="mail" uxBadge="NEW" badgeVerticalPosition="above" badgeHorizontalPosition="after"
+                badgeSize="{{ size }}" badgeColor="critical"></ux-icon>
```

Comment:
Just noticed there are a few places in this file where we are using `{{ }}` for attributes when we should use property binding instead eg: `[badgeSize]`

Reply:
Sorry good spot. That's fixed now.